CMakeLists.txt

```diff
@@ -77,6 +77,12 @@ if(NOT ONNX_NAMESPACE)
   set(ONNX_NAMESPACE "onnx")
 endif()
 
+# find_package Python has replaced PythonInterp and PythonLibs since cmake 3.12
+# Use the following command in the future; now this is only compatible with the latest pybind11
+# find_package(Python ${PY_VERSION} COMPONENTS Interpreter DevelopmentREQUIRED)
+find_package(PythonInterp ${PY_VERSION} REQUIRED)
```

Comment:
It depends on which cmake version you want to use.
For older cmake versions, please see how onnxruntime is handling how.  https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/templates/set-python-manylinux-variables-step.yml

For newer ones, please read https://gitlab.kitware.com/cmake/cmake/-/issues/20425